cartridges/int_algolia/cartridge/scripts/algolia/lib/algoliaData.js

```diff
@@ -57,18 +57,19 @@ const clientSideData = {
 
 //  Script for getting Algolia preferences
 //
-//             ID           ║                    description                          ║ type of preference
-//   ═══════════════════════╬═════════════════════════════════════════════════════════╬═══════════════════════════
-//   Enable                 ║ Enable/disable all Algolia                              ║ Boolean
-//   ApplicationID          ║ Identifies the application for this site                ║ String
-//   SearchApiKey           ║ Authorization key for Algolia                           ║ String
-//   AdminApiKey            ║ Authorization Admin key for Algolia                     ║ String
-//   AdditionalAttributes   ║ Any additional Product attributes                       ║ Set-of-string
-//   InStockThreshold       ║ Stock Threshold                                         ║ Double
-//   IndexPrefix            ║ Optional prefix for the index name                      ║ String
-//   EnableInsights         ║ Enables insights events on the Storefront               ║ Boolean
-//   EnableSSR              ║ Enables server-side rendering of CLP results on SFRA    ║ Boolean
-//  ════════════════════════╩═════════════════════════════════════════════════════════╩═══════════════════════════
+//             ID                 ║                    description                          ║ type of preference
+//   ═════════════════════════════╬═════════════════════════════════════════════════════════╬════════════════════
+//   Enable                       ║ Enable/disable all Algolia                              ║ Boolean
+//   ApplicationID                ║ Identifies the application for this site                ║ String
+//   SearchApiKey                 ║ Authorization key for Algolia                           ║ String
+//   AdminApiKey                  ║ Authorization Admin key for Algolia                     ║ String
+//   AdditionalAttributes         ║ Any additional Product attributes                       ║ Set-of-string
+//   AdditionalContentAttributes  ║ Any additional Content attributes                       ║ Set-of-string
```

Comment:
As decided, let's put these on job step level

Reply:
yes it seems I forget to update comment section, thanks